### PR TITLE
Update and format the README file

### PR DESCRIPTION
**Pull request by shavidissa**
Update and format the README file.

### CODE REVIEW

**Comment by shavidissa** [comment]
CC @rkempf42

File: README.md [hunk 1]
```diff
@@ -1,39 +1,154 @@
...
+* [Prerequisites](#Prerequisites)
+* [Set Up an IWavefrontSender](#set-up-an-iwavefrontsender)
+* [Send Data to Wavefront](#send-data-to-wavefront)
+* [Close the IWavefrontSender](#close-the-iwavefrontsender)
+* [License](#License)
+* [How to Get Support and Contribute](#how-to-get-support-and-contribute)
+
+# Welcome to the Wavefront C# SDK
+
+Wavefront by VMware C# SDK lets you send raw data from your .NET application to Wavefront an `IWavefrontSender` interface. The data is then stored as metrics, histograms, and trace data. This SDK is also called the Wavefront Sender SDK for C#.
```
Comment (rkempf42) [inline README.md]: Can't parse "...to Wavefront an `IWavefrontSender` interface.

File: README.md [hunk 2]
```diff
@@ -1,39 +1,154 @@
...
+
+![C# Sender SDK Decision Tree](docs/csharp_sender_sdk.png)
+
+> ***Note***:
+> </br>
+>   * **This is the Wavefront by VMware SDK for C# (Wavefront Sender SDK for C#)!**
+>   If this SDK is not what you were looking for, see the [table](#wavefront-sdks) below.
+
+
+#### Wavefront SDKs
```
Comment (rkempf42) [inline README.md:27]: Should we start removing the Framework SDKs? Or only from doc?

File: README.md [hunk 3]
```diff
@@ -60,8 +175,9 @@ wfProxyClientBuilder.FlushIntervalSeconds(2);
 IWavefrontSender wavefrontSender = wfProxyClientBuilder.Build();
 ```
 
-### Create a WavefrontDirectIngestionClient
-To create a `WavefrontDirectIngestionClient`, you must have access to a Wavefront instance with direct data ingestion permission:
+### Option 2: Sending Data via Direct Ingestion
+To create a `WavefrontDirectIngestionClient`, you must have access to a Wavefront instance with [direct data ingestion permission](https://docs.wavefront.com/permissions_overview.html):
```
Comment (rkempf42) [inline README.md]: Mmm. Maybe we need to mention Proxies permission above? 
And, I would capitalize Direct Data Ingestion permission (that's the name in the UI)

**Review by rkempf42 — COMMENTED** [review]
LGTM! A few very minor comments

File: README.md (hunk 2 above)
Comment (shavidissa) [reply README.md:27]: We thought we will leave the framework SDKs on the READMEs for now. But Once we remove it from docs we will need to think about some broken links here. :)

File: README.md (hunk 3 above)
Comment (shavidissa) [reply README.md]: Thank you! Good point!

**Comment by shavidissa** [comment]
Thank you for the review, @rkempf42! :)

**Review by rkempf42 — APPROVED** [review]
LGTM!

**Comment by shavidissa** [comment]
CC @sushantdewan123

**Review by sushantdewan123 — APPROVED** [review]

**Pull request merged**